### PR TITLE
Add vertical grid file for Belgian geoid model hBG18

### PR DESCRIPTION
**Pull request by AlexBass05**
The grid file allows converting from ellipsoidal height
in ETRS89 (EPSG:4937) to orthometric height in
Ostend height (EPSG:5710) using be_ign_hBG18.tif

The GeoTIFF file was created with the attached `build_hGB18.sh` script.

Corresponding PR in PROJ: https://github.com/OSGeo/PROJ/pull/3044

Created as a draft PR as it requires an EPSG database update to include this change request: https://epsg.org/closed-change-request/browse/id/2022.007

### CODE REVIEW

**Comment by rouault** [comment]
The validation script displays
```
The following warnings were found (the file will probably be read correctly by PROJ, but corrective action may be required):
 - Geotransform with a positive pixel height, that is a south-up image, is supported, but a unusual formulation
```
This warning could be avoided by first doing a "gdalwarp hBG18.dat temp.tif" that would produce a "north-up" raster, and then using vertoffset_grid_to_gtiff on it to add the tags.

**Comment by AlexBass05** [comment]
> The validation script displays
> 
> ```
> The following warnings were found (the file will probably be read correctly by PROJ, but corrective action may be required):
>  - Geotransform with a positive pixel height, that is a south-up image, is supported, but a unusual formulation
> ```
> 
> This warning could be avoided by first doing a "gdalwarp hBG18.dat temp.tif" that would produce a "north-up" raster, and then using vertoffset_grid_to_gtiff on it to add the tags.

Thanks for pointing this out. 
I regenerated the GeoTIFF (applying your suggestion) and ran `regenerate_index_html.py` again.

**Comment by rouault** [comment]
> > 
> 
> Thanks for pointing this out.

ok, actually I missed that the original raster didn't have square pixels, and that gdalwarp produces square pixels by default, so you have to invoke it with more arguments, something

like "gdalwarp hBG18.dat temp.tif -overwrite -tr 0.015 0.010 -te 0.9925  52.505 7.0075 48.495"
you should check that the output of gdalinfo on the original and target grid results in the same reported extent, resolution and dimensions.
and with the gdalsrsinfo utility on a few sample points on the original and resulting grid that you get the same values
like "gdallocationinfo -geoloc be_ign_hBG18.tif 4 50"

File: copyright_and_licenses.csv [hunk 1]
```diff
@@ -19,6 +19,7 @@ au_icsm_GDA94_GDA2020_conformal.tif,Intergovernmental Committee on Surveying and
 au_icsm_National_84_02_07_01.tif,Intergovernmental Committee on Surveying and Mapping (ICSM),CC-BY-4.0,,
 au_icsm_README.txt,Disclaimed,Public domain,,
 be_ign_bd72lb72_etrs89lb08.tif,Insititut Geographique National (IGN) Belgium,CC-BY-4.0,,
+be_ign_hBG18.tif,Insititut Geographique National (IGN) Belgium,CC-BY-4.0,1.9,
```
Comment (florisvdh) [inline copyright_and_licenses.csv:22]: Minor typo here: `Insititut` should be `Institut`

File: copyright_and_licenses.csv [hunk 2]
```diff
@@ -19,6 +19,7 @@ au_icsm_GDA94_GDA2020_conformal.tif,Intergovernmental Committee on Surveying and
 au_icsm_National_84_02_07_01.tif,Intergovernmental Committee on Surveying and Mapping (ICSM),CC-BY-4.0,,
 au_icsm_README.txt,Disclaimed,Public domain,,
 be_ign_bd72lb72_etrs89lb08.tif,Insititut Geographique National (IGN) Belgium,CC-BY-4.0,,
```
Comment (florisvdh) [inline copyright_and_licenses.csv:21]: Seems to be inherited from the line above, same typo.

File: copyright_and_licenses.csv (hunk 1 above)
Comment (florisvdh) [reply copyright_and_licenses.csv:22]: And `Geographique` should have an accent in both lines: `Géographique`, given that the French name is used :wink:

Comment (rouault) [reply copyright_and_licenses.csv:22]: fixed

**Pull request merged**